src/app/index.tsx

```diff
@@ -83,9 +84,11 @@ export default (options: Options) => {
   });
 
   const container = () => <Provider store={store}>
-    <Services.Provider value={services} >
-      <navigation.components.NavigationProvider routes={routes} />
-    </Services.Provider>
+    <MessageProvider>
```

Comment:
if MessageProvider is here it doesn't need to also be in SkipToContentWrapper right?

Reply:
Yes, I can change that. I was debating whether it made more sense to wrap only components that need the MessageProvider with it or to have a global Provider.

Reply:
i think providing it once globally is a little better, i could see the argument to applying it only to individual components though